cpp/tests/rolling/rolling_test.cpp

```diff
@@ -581,7 +581,7 @@ TYPED_TEST(RollingTest, ZeroWindow)
 {
   size_type num_rows = 1000;
 
-  std::vector<TypeParam> col_data(num_rows, 1);
+  std::vector<TypeParam> col_data(num_rows, TypeParam(1));
```

Comment:
wouldn't you have to include the duration types for this typed test for these changes to be necessary?

Reply:
Merging removed it. will add it again.

Reply:
Added.